From 429b2ba70812fc8ce7c591e787ec0f2b48d13319 Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.king@canonical.com>
Date: Thu, 3 Jun 2021 11:33:49 +0100
Subject: [PATCH] EDAC/mce_amd: Fix typo "FIfo" -> "Fifo"
Git-commit: 429b2ba70812fc8ce7c591e787ec0f2b48d13319
Patch-mainline: v5.14-rc1
References: bsn#84
Modified-by-SEL: No


There is an uppercase letter I in one of the MCE error descriptions
instead of a lowercase one. Fix it.

Signed-off-by: Colin Ian King <colin.king@canonical.com>
Signed-off-by: Borislav Petkov <bp@suse.de>
Reviewed-by: Yazen Ghannam <yazen.ghannam@amd.com>
Link: https://lkml.kernel.org/r/20210603103349.79117-1-colin.king@canonical.com
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 drivers/edac/mce_amd.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/edac/mce_amd.c b/drivers/edac/mce_amd.c
index 43ba0f931629..27d56920b469 100644
--- a/drivers/edac/mce_amd.c
+++ b/drivers/edac/mce_amd.c
@@ -431,7 +431,7 @@ static const char * const smca_xgmipcs_mce_desc[] = {
 	"Replay Buffer Parity Error",
 	"Data Parity Error",
 	"Replay Fifo Overflow Error",
-	"Replay FIfo Underflow Error",
+	"Replay Fifo Underflow Error",
 	"Elastic Fifo Overflow Error",
 	"Deskew Error",
 	"Flow Control CRC Error",
-- 
2.26.2

